app/tournament/match.js

```diff
@@ -0,0 +1,39 @@
+export class Match {
+    constructor(round) {
+        this.round = round;
+        this.players = [];
+        this.isFinished = false;
+    }
+
+    addPlayer(player) {
+        if (this.players.length >= 2) {
+            throw new Error("Match has already 2 players");
+        }
+        this.players.push(player);
+    }
+
+    setNextMatch(match) {
+        this.nextMatch = match;
+    }
+
+    setWinner(player) {
```

Comment:
It would be very natural to get a full player as parameter, but it would also mean a loss of readability in case you don't know which players there are. It would be necessary then to:
1. Check if player is in that match, comparing the name of each one, and in case not, throw.
2. Retrieve the player from the match's players array by means of the index and then call the method with it.
Since there are always 2 players in a match to be set as winner and they are already stored in the match, I found it more important here to simplify code and implementation by renaming the variable and using an index 0-1.